README.md

```diff
@@ -25,3 +19,7 @@ to be installed first.
 ## Quickstart
 
 See the [Quickstart](https://docs.tiledb.com/cloud/quickstart) section of the docs.
+
+## Developer links
+
+See vignettes in the Articles section.
```

Comment:
@eddelbuettel this was previously linking to `./HowToX.md`.

Now with those in `vignettes/HowToX.md` (thank you!), their HTML is in `docs/articles/HowToX.html`.

Given the path change from `vignettes` at the `.md` level to `articles` at the `.html` level I didn't see a way for links to work for markdown & HTML views both.

However I realize that the bullet-list is now redundant, since there is an entire tab devoted to articles. So I thought it would be simpler to remove the bullet-list entirely and simply point to the Articles tab.

Reply:
Yes that is better, as vignette support is baked in.